posthog/helpers/session_recording.py

```diff
@@ -183,12 +215,8 @@ def decompress_chunked_snapshot_data(
         # Decompressed data can be large, and in metadata calculations, we only care if the event is "active"
         # This pares down the data returned, so we're not passing around a massive object
         if return_only_activity_data:
-            events_with_only_activity_data = [
-                {"timestamp": recording_event.get("timestamp"), "is_active": is_active_event(recording_event)}
-                for recording_event in decompressed_data
-            ]
+            events_with_only_activity_data = get_events_summary_from_snapshot_data(decompressed_data)
```

Comment:
I think this introduces new code that now loops through all snapshot data on every session recording fetch which could potentially make first time to paint slower. Did you notice any performance hits for recordings with large numbers of snapshots?

Reply:
I am fairly sure it was looping through all the decompressed_data before as well so should be the same.